docs/src/components/DocLayout/index.tsx

```diff
@@ -158,7 +168,18 @@ export default function DocLayout({
                         {tabs && <Tabs activeTab={location.pathname} tabs={tabs} />}
                         {headerLink && (
                           <Hide on={["smallMobile", "mediumMobile"]}>
-                            <HeaderButtonLink href={headerLink} />
+                            <Stack flex spacing="small">
```

Comment:
The spacing is too big, I'd remove it completely because `HeaderButtonLink` already provides a lot of spacing. Also the label "Storybook" might be useful next to the Storybook icon. I'd ask @williamkolmacka for guidance here.

Reply:
do we really need that padding inside `HeaderLink`?  it looks better without.